### PR TITLE
EN_REboot: Some capitalization improvements and obvious word translations

### PR DESCRIPTION
**Pull request by thelastomen**

### CODE REVIEW

File: Translated/Tasks/Daily/ed0610010320.csv [hunk 1]
```diff
@@ -1,3 +1,3 @@
 Title#0,"""Procure Northern Aerio Ore"""
 Summary#0,"""Obtain Ore by Gathering at Northern Aerio<br><br><yellow>※Target materials:<br>\u3000Trinite, Photon Quartz<c>"""
-text0101#0,"""北エアリオで鉱石を入手する"""
+text0101#0,"""Gather ores at North Aerio"""
```
Comment (DabirSA) [inline Translated/Tasks/Daily/ed0610010320.csv]: Northern Aerio

**Review by DabirSA — CHANGES_REQUESTED** [review]

File: Translated/Tasks/Daily/ed0610010320.csv (hunk 1 above)
Comment (thelastomen) [reply Translated/Tasks/Daily/ed0610010320.csv]: I have updated the text now. thanks!

File: Translated/Tasks/Side/ss0030766051.csv [hunk 2]
```diff
@@ -1,4 +1,4 @@
 Title#0,"""Making Multi-Weapons"""
-Summary#0,"""Staggy, an ARKS member,<br>teaches you how to make Multi-Weapons<br>at the Item Lab.<br><br><yellow>※Recommended Gathering Sections：<br>Nothern Aerio<c><c>"""
+Summary#0,"""Staggy, an ARKS member,<br>teaches you how to make Multi-Weapons<br>at the Item Lab.<br><br><yellow>※Recommended Gathering Sections：<br>Northern Aerio<c><c>"""
```
Comment (DabirSA) [inline Translated/Tasks/Side/ss0030766051.csv]: Double <c> tag here. Shouldn't cause problems but not necessary

Comment (thelastomen) [reply Translated/Tasks/Side/ss0030766051.csv]: do you refer to the double line break or the \<c\> tag here?

Comment (DabirSA) [reply Translated/Tasks/Side/ss0030766051.csv]: The \<c\> at the end. Sorry, didn't realise that wasn't visible in the comment.

**Review by DabirSA — APPROVED** [review]

**Pull request merged**